funnel/templates/schedule.html.jinja2

```diff
@@ -112,7 +112,7 @@
                   <h2 class="mui--text-heading schedule__date">{{ schedule.dateStr }}</h2>
                   <div class="mui--bg-accent schedule__table">
                     <div class="schedule__row schedule__row--sticky">
-                      <p class="schedule__row__column schedule__row__column--time schedule__row__column--time--header" :style="{ minWidth: timeSlotWidth + 'px'}" v-if="(view == 'calendar')"><svg class="fa5-icon fa5-icon--subhead test fa5--align-baseline" aria-hidden="true" role="img"><use v-bind:xlink:href="svgIconUrl + '#clock'"></use></svg></p>
+                      <p class="schedule__row__column schedule__row__column--time schedule__row__column--time--header" :style="{ minWidth: timeSlotWidth + 'px'}" v-if="(view == 'calendar')"><fa-svg :icon="'clock'" :iconsize="'subhead'"></fa-svg></p>
```

Comment:
Here it is `<fa-svg>` instead of `<faSvg>`. This is confusing.

Reply:
Changed for both vue and ractive component to faicon and user_avatar